src/Squidex.Domain.Apps.Entities/Contents/ContentOperationContext.cs

```diff
@@ -106,7 +107,20 @@ public Task EnrichAsync()
 
         private async Task<IReadOnlyList<IAssetInfo>> QueryAssetsAsync(Guid appId, IEnumerable<Guid> assetIds)
         {
-            return await assetRepository.QueryAsync(appId, null, new HashSet<Guid>(assetIds), null, int.MaxValue, 0);
```

Comment:
Please use this Method instead:

Task<IResultList<IAssetEntity>> QueryAsync(Guid appId, HashSet<Guid> ids);

A little bit stupid to format to odata first and then parse it again ;)